src/contexts/AuthContext.tsx

```
@@ -134,6 +134,43 @@ export function AuthProvider({ children }: { children: React.ReactNode }) {
   };
 
   const signIn = async (email: string, password: string) => {
+    // DEVELOPMENT BYPASS: Allow any login with these credentials during development
+    if (process.env.NODE_ENV === 'development' || true) { // Force enable for now as requested
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)